CHANGE-NOTES.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## v3.0.3
+- Enabled Ctrl + mouse click to work as a right-click for Mac users
```

Comment:
```suggestion
- Enabled Ctrl + left-click to work as a right-click for Mac users, for displaying the machete actions on a branch.
```
left click can happen both on the Touch Bar and with mouse.

Reply:
👍